docs/en/concept/schema-evolution.md

```diff
@@ -43,9 +52,8 @@ source {
     password = "mysqlpw"
     table-names = ["shop.products"]
     base-url = "jdbc:mysql://mysql_cdc_e2e:3306/shop"
-    debezium = {
-      include.schema.changes = true
-    }
```

Comment:
The old version config can not work well now?

Reply:
Compatible with older versions

https://github.com/apache/seatunnel/pull/8252/files#diff-a245700d76fc875d2c1405fd081a42b81c18359a2ec8e4572ff6e1e8f7562d42R81
https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/config/MySqlSourceConfigFactory.java#L110